### ISSUE TITLE
Atuin has changed URL

### ISSUE THREAD

**Issue by danielmichaels** [alexellis/arkade#1092]
## Expected Behaviour

`arkade get atuin` should successfully download the binary.

## Current Behaviour

The response is now a 404

```shell
z ❯ ark get atuin
Downloading: atuin
2024/06/19 10:27:25 Looking up version for atuin
2024/06/19 10:27:25 Found: v18.3.0
Downloading: https://github.com/atuinsh/atuin/releases/download/v18.3.0/atuin-v18.3.0-x86_64-unknown-linux-gnu.tar.gz

The requested version of atuin is not available or configured in arkade for Linux/x86_64

* Check if a binary is available from the project for your Operating System
* View the atuin releases page: https://github.com/atuinsh/atuin/releases
* Feel free to raise an issue at https://github.com/alexellis/arkade/issues for help

Error: server returned status: 404
```

## Possible Solution

The author has changed the path for releases.

`arkade` is using:

- `https://github.com/atuinsh/atuin/releases/download/v18.3.0/atuin-v18.3.0-x86_64-unknown-linux-gnu.tar.gz`

But the new path is:

- `https://github.com/atuinsh/atuin/releases/download/v18.3.0/atuin-x86_64-unknown-linux-gnu.tar.gz`

It appears they've removed the version from the tar file names. This change looks like it starts from `v18.3.0` upwards. 

Releases pages for reference:

- 18.2.0 [release](https://github.com/atuinsh/atuin/releases/tag/v18.2.0) page
- 18.3.0 [release](https://github.com/atuinsh/atuin/releases/tag/v18.3.0) page

## Steps to Reproduce (for bugs)

1. `arkade get atuin` (404)
2. `arkade get atuin --version=v18.2.0` (200 success)

## Context

This issue prevents the installation of versions of [atuin](https://github.com/atuinsh/atuin) after `v18.2.0`.

## Your Environment

Ubuntu 22.04

```
# uname -a
Linux P91F003 6.5.0-35-generic #35~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue May  7 09:00:52 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

# cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

* What arkade version is this?

```
# arkade version
z ❯ ark version
            _             _
  __ _ _ __| | ____ _  __| | ___
 / _` | '__| |/ / _` |/ _` |/ _ \
| (_| | |  |   < (_| | (_| |  __/
 \__,_|_|  |_|\_\__,_|\__,_|\___|

Open Source Marketplace For Developer Tools

Version: 0.11.6
Git Commit: acb1bd999ac81840fbbe49f3d1cc1fbd386f7fd4
```

## Additional comments

I am raising this for awareness but also happy to make a PR for it as well.

**Comment by rgee0** [comment]
Your version: 0.11.6
Current version:0.11.16

Atuin was fixed in [0.11.15](https://github.com/alexellis/arkade/releases/tag/0.11.15) by https://github.com/alexellis/arkade/commit/e4283fd0fd086a1fca574d0136ecdb26e6589869

Please update your version of arkade and retry.

**Comment by danielmichaels** [comment]
My mistake! I saw 0.11.6 and thought it was 0.11.16, closing. 

Incidentally, I did run `arkade get arkade` whilst doing this issue but did not notice that it failed to download the latest version (probably how I have my machine configured). Instead, today I ran `arkade update` and it succeeded. Noted for future that is how I should update the binary.

Confirmed working on 0.11.16

Apologies for wasting your time
